pkg/features/embedded/cloud-config-defaults/system/oem/04_accounting.yaml

```diff
@@ -14,4 +14,4 @@ stages:
          entity: |
             kind: "shadow"
             username: "root"
-            password: "cos"
+            password: "root"
```

Comment:
Nice :+1:

Reply:
I was also considering using `elemental` ;) I'd be fine with anything to be honest, no strong opinion at all

Reply:
let me put the password change in a different and independent PR ;)